docs/docs/self-host/env.mdx

```diff
@@ -364,3 +364,4 @@ Pass the JSON string into the environment variable SSO_CONFIG of your GrowthBook
 - **DISABLE_TELEMETRY** - We collect anonymous telemetry data to help us improve GrowthBook. Set to "true" to disable.
 - **STORE_SEGMENTS_IN_MONGO** - If using the config.yml file, set to `true` if you want to store segments in Mongo. This is also useful if you have existing segments stored in Mongo that you need to access. When set, GrowthBook will ignore segments in the config.yml file and only use Segments stored in Mongo.
 - **CDN_HOST** - When set, this will update the implementation instructions within GrowthBook to override the `API_HOST` in cases where a CDN is used.
+- **EXPRESS_TRUST_PROXY_OPTS** - Express' [trust proxy](https://expressjs.com/en/5x/api.html#trust.proxy.options.table) setting value. Supports boolean (true/false) and string values. Leavy empty or specify `false` to use Express' default behavior.
```

Comment:
Should we update this to say we support integer values as well?  Also, might be worth adding a sentence saying why you would set this.  Something like:
> If you are running GrowthBook behind a proxy or load balancer, this is required to track the correct user ip for audit log events

Reply:
Yes I can add that.

Reply:
Updated in [77b78e4](https://github.com/growthbook/growthbook/pull/1149/commits/77b78e46a3996439a4d57e2a263441681e980365)